Grundgeruest/models.py

```diff
@@ -210,25 +211,43 @@ class ScholariumProfile(UserenaBaseProfile):
     alt_registration_ip = models.GenericIPAddressField(
         editable=False, null=True)
 
+    def get_unterstuetzungen(self):
+        '''Gibt alle bisherigen Unterstützungen zurück.'''
+        return Unterstuetzung.objects.filter(profil=self)
```

Comment:
die db legt automatisch schon eine backreference an; der default Name ist glaube ich self.unterstuetzung, man könnte den aber auch in dem profil-Feld im Unterstützungs-Model setzen (mit reverse_name='irgendwas')
Ist aber nur eine performance-Frage...